assets/src/edit-story/output/page.js

```diff
@@ -59,21 +64,19 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
         <amp-story-grid-layer template="vertical">
           <div className="page-background-area" style={backgroundStyles}>
             {backgroundElements.map((element) => (
-              <OutputElement key={'el-' + element.id} element={element} />
+              <OutputElement key={element.id} element={element} />
+            ))}
+            {backgroundOverlays.map((element) => (
+              <div
+                key={element.id}
```

Comment:
This looks to be making duplicate keys. Not super critical for output, but perhaps we should still give them prefix?

Reply:
Keys are a react thing, they are never going in the DOM. And React keeps track of the keys of the items of each list separately. So there's nothing to worry about here.

Reply:
Sure. But React expects them to be unique across the children of a component, and in this case they are not?

Reply:
I'm pretty sure they only have to be unique within their respective arrays. After JSX this become 

```js
React.createElement('div', {attrs}, [{key: 1, ....}, {key: 2, ....}], [{key: 1, ....}, {key: 2, ....}]);
```

So React knows fully well, that each array is a separate set of children - and only uses the keys within each array to make sure they are reused when relevant. I could be wrong here, but I've never seen any errors with this.

See e.g. [this example](https://codepen.io/barklund/pen/RwWvBWx?editors=0010) modified from [this React guide](https://reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings) where no warnings are given in my pen even though the keys of the two arrays are "direct" siblings (but in separate lists of nodes).

Reply:
oh yeah. that makes sense. thanks!